cmapi/mcs_cluster_tool/cluster_app.py

```diff
@@ -32,9 +44,119 @@ def status():
 
 @app.command()
 @handle_output
-def stop():
+@TransactionManager(
+    timeout=timedelta(days=1).total_seconds(), handle_signals=True
+)
+def stop(
+    interactive: Annotated[
+        bool,
+        typer.Option(
+            '--interactive/--no-interactive', '-i/-no-i',
+            help=(
+                'Use this option on a highly loaded cluster. '
+                'It prevents data loss.'
+            ),
+        )
+    ] = False,
+    timeout: Annotated[
+        int,
+        typer.Option(
+            '-t', '--timeout',
+            help=(
+                'Time in seconds to wait DMLproc gracefully stops. '
+                'Use this with caution: on a working cluster '
+                'low values could cause data loss. '
+                'In interactive mode means delay time between promts.'
+            )
+        )
+    ] = 15,
+    force: Annotated[
+        bool,
+        typer.Option(
+            '--force/--no-force', '-f/-no-f',
+            help=(
+                'Force stopping a cluster.'
```

Comment:
Idea:
Force stops columnstore
Warning: This could cause data corruption and/or data loss

Reply:
Done.